addons/sale_timesheet/tests/test_project_billing.py

```diff
@@ -417,15 +415,15 @@ def test_customer_change_in_project(self):
             'partner_id': self.partner_2.id,
         })
         self.assertFalse(self.project_project_rate.sale_order_id, "The SO in the project should be False because the previous SO does not for the actual customer of the project.")
-        self.assertFalse(self.project_project_rate.sale_line_id, "The SOL in the project should be False because the SO is removed too.")
+        self.assertFalse(self.project_project_rate.sale_line_id, "The SOL in the project should be False because the previous SOL does not for the actual customer of the project.")
```

Comment:
I do not get the sense of the sentence. I see you just used the same as above but I do not get the sense of it too 🙂 
"... does not match with the current customer"?

Reply:
Because the SO is now a related field in project.project model. Before this old sentence means when the customer is changed in the current project, the SO is removed and then a compute is triggered for the SOL to remove it too.
Since the SO is a related field, the SOL is removed when the user changes the customer in the project and no longer because of the SO is removed.
That's why I change the sentence.